component-registry.json

```
@@ -0,0 +1,27 @@
+[
+    {
+        "name": "Weather Server",
+        "description": "A weather component written in JavaScript",
+        "oci_uri": "oci://ghcr.io/microsoft/get-weather-js:latest"
```

```suggestion
        "uri": "oci://ghcr.io/microsoft/get-weather-js:latest"
```

Since the schema "oci" is implied in the "oci://..." itself.